prow/lib.sh

```diff
@@ -195,7 +203,7 @@ EOF
     apiVersion: kind.sigs.k8s.io/v1alpha3
     networking:
       podSubnet: 10.20.0.0/16
-      serviceSubnet: 10.255.10.0/24
+      serviceSubnet: 10.255.20.0/24
```

Comment:
it was using the same subnet ip for both clusters, for inter-cluster service connectivity subnets should not overlap

Reply:
thanks. fyi @nmittler